pkg/controller/clusterdeployment/clusterdeployment_controller.go

```diff
@@ -746,20 +746,24 @@ func (r *ReconcileClusterDeployment) updateClusterDeploymentStatus(cd *hivev1.Cl
 
 	// The install manager sets this secret name, but we don't consider it a critical failure and
 	// will attempt to heal it here, as the value is predictable.
-	if cd.Status.Installed && cd.Status.AdminKubeconfigSecret.Name == "" {
-		cd.Status.AdminKubeconfigSecret = corev1.LocalObjectReference{Name: fmt.Sprintf("%s-admin-kubeconfig", cd.Name)}
+	if cd.Status.Installed && cd.Status.RawAdminKubeconfigSecret.Name == "" {
+		cd.Status.RawAdminKubeconfigSecret = corev1.LocalObjectReference{Name: fmt.Sprintf("%s-admin-kubeconfig", cd.Name)}
 	}
```

Comment:
Do you still need the workaround we talked about for old install manager images? Those will set AdminKubeconfigSecret instead of Raw, if we see that set but no Raw we need to copy it to raw and then update AdminKubeconfigSecret.

Reply:
For old install manager images, the RawAdminKubeconfigSecret won't be set, so we'll set it with the code above. After that, the flow is with the new install manager image.